.github/workflows/continuous-delivery.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3.5.1
         with:
-          node-version: 14
+          node-version: 16
       - name: Cache dependencies
```

Comment:
nit: setup-node action has a built-in cache for npm for a while, so the `Cache dependencies` step can be removed in favor of a one-liner [like this](https://github.com/ridedott/portal/blob/29197c6e9098146481924e72eb268af7496c08e5/.github/workflows/continuous-integration.yaml#L33)

Reply:
Done!